final class SocketException implements Exception {
  final String reason;

  const SocketException({required this.reason});
}
